agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -99,7 +99,7 @@ const (
 	StatsInactivityTimeout = 5 * time.Second
 
 	// retry settings for pulling images
-	maximumPullRetries        = 10
+	maximumPullRetries        = 5
```

Comment:
Does it have any value to make this configurable?

Reply:
As discussed offline, currently we don't see value in making it configurable but can be done in future if users see value in it.